frontend/src/model/beta/releases/EditableRelease.tsx

```diff
@@ -0,0 +1,138 @@
+import { Box, Typography } from '@mui/material'
+import { useGetModel } from 'actions/model'
+import { postFile, putRelease, UpdateReleaseParams, useGetReleasesForModelId } from 'actions/release'
+import { useCallback, useContext, useEffect, useState } from 'react'
+import Loading from 'src/common/Loading'
+import UnsavedChangesContext from 'src/contexts/unsavedChangesContext'
+import EditableFormHeading from 'src/Form/beta/EditableFormHeading'
+import MessageAlert from 'src/MessageAlert'
+import ReleaseForm from 'src/model/beta/releases/ReleaseForm'
+import { FlattenedModelImage } from 'types/interfaces'
+import { ReleaseInterface } from 'types/types'
+import { getErrorMessage } from 'utils/fetcher'
+
+type EditableReleaseProps = {
+  release: ReleaseInterface
+}
+
+export default function EditableRelease({ release }: EditableReleaseProps) {
+  const [isEdit, setIsEdit] = useState(false)
+  const [semver, setSemver] = useState(release.semver)
+  const [releaseNotes, setReleaseNotes] = useState(release.notes)
+  const [isMinorRelease, setIsMinorRelease] = useState(!!release.minor)
+  const [artefacts, setArtefacts] = useState<File[]>([]) // TODO - Default to using the release artefact files (BAI-1026)
+  const [imageList, setImageList] = useState<FlattenedModelImage[]>(release.images)
+  const [errorMessage, setErrorMessage] = useState('')
+  const [isLoading, setIsLoading] = useState(false)
+
+  const { model, isModelLoading, isModelError } = useGetModel(release.modelId)
+  const { mutateReleases } = useGetReleasesForModelId(model?.id)
+
+  const { setUnsavedChanges } = useContext(UnsavedChangesContext)
+
+  const resetForm = useCallback(() => {
+    setSemver(release.semver)
+    setReleaseNotes(release.notes)
+    setIsMinorRelease(!!release.minor)
+    setArtefacts([]) // TODO - Reset the release artefact files (BAI-1026)
+    setImageList(release.images)
+  }, [release.images, release.minor, release.notes, release.semver])
+
+  useEffect(() => {
+    resetForm()
+  }, [resetForm])
+
+  useEffect(() => {
+    setUnsavedChanges(isEdit)
+  }, [isEdit, setUnsavedChanges])
+
+  if (isModelError) {
+    return <MessageAlert message={isModelError.info.message} severity='error' />
```

Comment:
Did we decide on how we wanted to handle errors? Happy to leave this for another ticket if that's what we decided

Reply:
Yeah this is what we decided on. Any components that don't include the `Wrapper` can safely use a `MessageAlert`. Any of the page components with their own `Wrapper` will need to use the `MultipleErrorWrapper`